File: examples_android/app/src/main/java/com/confirmit/testsdkapp/layouts/defaultlayout/items/DefaultLayoutClickLabel.kt
package com.confirmit.testsdkapp.layouts.defaultlayout.items

import com.confirmit.mobilesdk.ui.questions.QuestionAnswer

enum class DefaultLayoutClickLabelType {
    RADIO,
    CHECKBOX
}

class DefaultLayoutClickLabel(val clickLabelType: DefaultLayoutClickLabelType, val initial: Boolean, val indent: Int, val answer: QuestionAnswer) : DefaultLayoutItem

File: examples_ios/MyApp/Sources/SurveyLayouts/DefaultLayout/Items/DefaultLayoutClickLabel.swift
import ConfirmitMobileSDK
import Foundation

enum DefaultLayoutClickLabelType {
  case radio
  case checkbox
}

class DefaultLayoutClickLabel: DefaultLayoutItem {
  let clickLabelType: DefaultLayoutClickLabelType
  let indent: Int
  let answer: QuestionAnswer

  init(type: DefaultLayoutClickLabelType, initial: Bool, indent: Int, answer: QuestionAnswer) {
    clickLabelType = type
    self.indent = indent
    self.answer = answer
  }
}
